test/integration/virtualnode/virtualnode_test.go

```diff
@@ -149,6 +161,101 @@ var _ = Describe("VirtualNode", func() {
 
 		})
 
+		It("should create a virtual node in CloudMap", func() {
```

Comment:
nit: should create a virtual node with CloudMap discovery

Reply:
Done